blockchain.go

```diff
@@ -158,6 +157,7 @@ type config struct {
 	TransactionValidationEnabled bool
 	ChainID                      flowgo.ChainID
 	CoverageReportingEnabled     bool
+	WithContracts                []ContractDescription
```

Comment:
Maybe drop the With prefix, or rename to something like InitialContracts, or StartupContracts, or just Contracts

Reply:
Updated in https://github.com/onflow/flow-emulator/pull/367/commits/b4ab4b80457b775c2c360df6bdc8cfdf98013044